data/hai/hai reveal 6 end.txt

```diff
@@ -423,14 +423,47 @@ mission "Hai Reveal [C05] Cleanup"
 		event "hai-human resolution announced"
 		fail "Hai Reveal [A11-A] Permanent Fleets"
 		fail "Hai Reveal [B02-A] The Blockade Goes Up"
+
+		# Delay after Hai Reveal before the Hai government releases Sayari, if they refused her before.
+		event "Sayari available after Hai Reveal" 5 15
+
 		conversation
+			# Make Danforth available if Hai Reveal has him reserved.
+			branch nodan
+				"Danforth in mission sequence" != 1
+			action
+				"Danforth in mission sequence" = 0
```

Comment:
```suggestion
				"Danforth in mission sequence" == 0
```
This is how it works, correct?

Reply:
No.

`a == b` means "does `a` equal `b`?"
`a = b` means "assign the value of `b` to `a`"

Endless Sky data files use C++ notation.

Reply:
I’m not used to C++ at all, so you know best. I was just guessing based on things in the Free Worlds files.